android/app/src/main/java/app/organicmaps/util/UiUtils.java

```diff
@@ -140,7 +140,7 @@ public static void visibleIf(boolean condition, View... views)
 
   public static void showIf(boolean condition, View view)
   {
-    view.setVisibility(condition ? View.VISIBLE : View.GONE);
+    view.setVisibility(condition ? View.VISIBLE : View.INVISIBLE);
   }
```

Comment:
it's due to this, this showIf function is used in lots of places, eg. t_secondary_title in the place page. Need to make a new function to switch between visible and invisible, and keep the original behaviour here.

Reply:
Ah oof sorry.
Will fix

Reply:
Should be fixed now, can you test @RedAuburn or @vng ? :)
My notebook hates me, so I couldn't test it, but will try further to comfirm

Reply:
working correctly here too :)